buildcontext/git.go

```diff
@@ -176,7 +176,7 @@ func (gr *gitResolver) resolveGitProject(ctx context.Context, gwClient gwclient.
 	if err != nil {
 		return nil, "", "", errors.Wrap(err, "failed to get url for cloning")
 	}
-	analytics.Count("gitResolver.resolveEarthProject", analytics.RepoHashFromCloneURL(gitURL))
+	analytics.Count("gitResolver.resolveEarthProject", "")
```

Comment:
Curious what does this mean (that's it's an empty string)?

Reply:
In this case, it just tracks usage of the remote reference feature, but without being specific.